bundles/org.openhab.core.config.discovery.mdns/src/main/java/org/openhab/core/config/discovery/mdns/internal/MDNSDiscoveryService.java

```diff
@@ -223,6 +240,10 @@ private void considerService(ServiceEvent serviceEvent) {
                         if (result != null) {
                             final DiscoveryResult resultNew = getLocalizedDiscoveryResult(result,
                                     FrameworkUtil.getBundle(participant.getClass()));
+                            final ServiceInfo service = serviceEvent.getInfo();
```

Comment:
**Naming:** We have `ServiceEvent` and `ServiceInfo`. To avoid confusion, I would suggest to stick to this naming for both variables, i.e. `serviceEvent` (already named that) and `serviceInfo`, respectively.

Reply:
Ok.